components/camel-google/camel-google-mail/pom.xml

```diff
@@ -83,6 +85,11 @@
             <artifactId>google-api-services-gmail</artifactId>
             <version>${google-api-services-mail-version}</version>
         </dependency>
+        <dependency>
```

Comment:
What is using guava?

Reply:
com.google.http-client:google-api-client:1.39.2 (his [pom.xml](https://github.com/googleapis/google-http-java-client/blob/main/google-http-client/pom.xml#L140))
With guava ver. 19.0 we have an error
<summary>Caused by: java.lang.NoSuchMethodError: com.google.common.io.ByteStreams.exhaust(Ljava/io/InputStream;)J</summary>
<details>
Caused by: java.lang.NoSuchMethodError: com.google.common.io.ByteStreams.exhaust(Ljava/io/InputStream;)J
        at com.google.api.client.http.ConsumingInputStream.close(ConsumingInputStream.java:40)
        at java.base/java.util.zip.InflaterInputStream.close(InflaterInputStream.java:232)
        at java.base/java.util.zip.GZIPInputStream.close(GZIPInputStream.java:137)
        at java.base/sun.nio.cs.StreamDecoder.implClose(StreamDecoder.java:378)
        at java.base/sun.nio.cs.StreamDecoder.close(StreamDecoder.java:193)
        at java.base/java.io.InputStreamReader.close(InputStreamReader.java:200)
        at com.google.gson.stream.JsonReader.close(JsonReader.java:1220)
        at com.google.api.client.json.gson.GsonParser.close(GsonParser.java:53)
        at com.google.api.client.json.JsonParser.parse(JsonParser.java:363)
        at com.google.api.client.json.JsonParser.parse(JsonParser.java:335)
        at com.google.api.client.json.JsonObjectParser.parseAndClose(JsonObjectParser.java:79)
        at com.google.api.client.json.JsonObjectParser.parseAndClose(JsonObjectParser.java:73)
        at com.google.api.client.http.HttpResponse.parseAs(HttpResponse.java:456)
        at com.google.api.client.auth.oauth2.TokenRequest.execute(TokenRequest.java:324)
        at com.google.api.client.googleapis.auth.oauth2.GoogleCredential.executeRefreshToken(GoogleCredential.java:384)
        at com.google.api.client.auth.oauth2.Credential.refreshToken(Credential.java:470)
        at org.apache.camel.component.google.mail.BatchGoogleMailClientFactory.authorizeServiceAccount(BatchGoogleMailClientFactory.java:89)
        at org.apache.camel.component.google.mail.BatchGoogleMailClientFactory.makeClient(BatchGoogleMailClientFactory.java:76)
        at org.apache.camel.component.google.mail.GoogleMailComponent.getClient(GoogleMailComponent.java:62)
        at org.apache.camel.component.google.mail.GoogleMailEndpoint.getClient(GoogleMailEndpoint.java:117)
        at org.apache.camel.component.google.mail.GoogleMailEndpoint.afterConfigureProperties(GoogleMailEndpoint.java:103)
        at org.apache.camel.support.component.AbstractApiEndpoint.configureProperties(AbstractApiEndpoint.java:138)
        at org.apache.camel.support.DefaultComponent.setProperties(DefaultComponent.java:425)
        at org.apache.camel.support.component.AbstractApiComponent.createEndpoint(AbstractApiComponent.java:90)
        at org.apache.camel.support.DefaultComponent.createEndpoint(DefaultComponent.java:171)
        at org.apache.camel.impl.engine.AbstractCamelContext.doGetEndpoint(AbstractCamelContext.java:934)
        at org.apache.camel.impl.engine.AbstractCamelContext.getEndpoint(AbstractCamelContext.java:850)
</details>

Reply:
Ah okay there is a 

google-mail-guava-version in parent/pom.xml that was previously in use. Can you update that to 20, and use this placeholder

Reply:
+1